d/dktest/dktest_ubi_8.3.sh

```diff
@@ -0,0 +1,72 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : dktest
+# Version               : 0.3.0,0.3.6
+# Source repo           : https://github.com/dhui/dktest
+# Tested on             : Ubuntu 20.04
+# Script License        : Apache License, Version 2 or later
+# Passing Arguments     : Passing Arguments: 1.Version of package,
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Arumugam N S<asellappen@yahoo.com>/Priya Seth<sethp@us.ibm.com>
+#
+# Disclaimer            : This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=master
```

Comment:
Can we have the default as latest tag (0.3.6??) instead of master ?

Reply:
done ,pl check